### DISCUSSION TITLE
Prettier and markdown plugins disagree about list marker spaces

### DISCUSSION THREAD

**Discussion by jpfender** [LazyVim/LazyVim#2015] [Q&A] [answered]
I have both `lazyvim.plugins.extras.formatting.prettier` and `lazyvim.plugins.extras.lang.markdown` enabled, which means that my md files are being formatted with prettier and linted with markdownlint. However, those two seem to disagree about list marker spaces. Prettier puts three spaces after each list marker, which markdownlint then complains about:

![Screenshot 2023-11-14 at 09 12 00](https://github.com/LazyVim/LazyVim/assets/685798/5c0b0d8e-98bb-4d2b-8cae-e5cddce75af5)

I'm wondering what would be the smartest way to resolve this conflict short of just ignoring the `list-marker-space` rule.

**Comment by dpetka2001** [comment, marked as answer]
According to the [docs](https://github.com/igorshubovych/markdownlint-cli#configuration) you could put a `.markdownlint.yaml` or corresponding `jsonc` if you prefer where you define how long the spaces should be. See [example](https://github.com/DavidAnson/markdownlint/blob/49ef661478bcf0a012f6dfe23872cea730a246f6/schema/.markdownlint.yaml#L135-L143) here.
